Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-90/00/00/1/translation.txt

```diff
@@ -1,11 +1,10 @@
+//
+// Dumped for AI Girl v1.2.3 by Text Dump v1.4.4
```

Comment:
Can you leave these comments out? They will create a lot of unnecessary noise in the diffs and when parsing these files with other tools.

Reply:
It's vital to my workflow, and being able to support them from my end. They only change when I start from a clean dump or add new files. In this case I started from a clean dump and re-applied everything because there were so many new items.

Reply:
They get cut out by most current translation/cleanup tools so there's a good chance they will disappear at a random point in the future.

Reply:
I'd really rather they weren't. I know they're stripped when the releases are made which is fine because the tree gets tagged at that point so it's easy enough to figure things out from there. Everyone has been good about keeping them until now, and I added them to TexDump.  There's a history of translation notes in comments so I figured they were preserved by most everything anyhow.

Reply:
It's still unnecessary noise in the diffs on every update. It's not possible to see what actually changed because the diff is so massive.

Reply:
Normally they don't change ever once added once , as I said above.  This dump was missing so much it was easier to take the clean dump and reapply the existing translations to it.  Having these lines made things a lot easier for me to keep on top of things, but I take your point that for files otherwise unchanged it's muddying up the review, so I'll have to rework how I'm doing things. It'll take me a bit so I'll just close the review for now